packages/sonataflow-quarkus-devui/sonataflow-quarkus-devui-deployment/pom.xml

```diff
@@ -64,7 +64,7 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-rest-client-deployment</artifactId>
+      <artifactId>quarkus-resteasy-client-deployment</artifactId>
```

Comment:
This is required when using RHBQ 3.8.4 (Quarkus product) to workaround a bug on their release.